packages/patternfly-4/react-core/src/components/TextInput/TextInput.tsx

```diff
@@ -114,10 +117,14 @@ class TextInputBase extends React.Component<TextInputProps> {
         disabled={isDisabled}
         readOnly={isReadOnly}
         ref={innerRef}
+        {...(ouiaContext.isOuia && {
+          'data-ouia-component-type': 'PF4/TextInput',
```

Comment:
The rest of the OUIA enabled components do not include the `PF4/` prefix. Why are you adding it for only TextInput?

Reply:
@redallen Actually I created a PR with changes for ouia enabled components https://github.com/patternfly/patternfly-react/pull/3622

Reply:
Very well. You'll have to handle not having the `PF4/` prefix for older versions in your QA suite.